sandbox-backend/src/main/java/com/exadel/sandbox/team5/util/Pair.java

```diff
@@ -1,14 +1,25 @@
 package com.exadel.sandbox.team5.util;
 
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
-import lombok.*;
-
-@Getter
 @Setter
-@NoArgsConstructor
-@AllArgsConstructor
+@Getter
 @EqualsAndHashCode
+@NoArgsConstructor
 public class Pair {
-    private long first;
-    private double second;
+    private String first;
+    private String second;
+
+    public Pair(long first, double second){
+        this.first=""+first;
+        this.second=""+second;
```

Comment:
Note for new PRs. It better to use `Double.toString(value)` or `Long.toString(value)` also don't forget about null values

Reply:
will make it by next PR for statistic